src/sentry/issues/grouptype.py

```diff
@@ -43,6 +70,62 @@ def __post_init__(self) -> None:
         if self.category not in valid_categories:
             raise ValueError(f"Category must be one of {valid_categories} from GroupCategory.")
 
+        # ensure that noise config ignore limits only increase with rollouts
+        # use a ratio of ignore limit to expiry time to allow for greater flexibility
+        if self.group_policy:
+            group_policy = self.group_policy
+
+            if group_policy.feature:
+                try:
+                    features.get(group_policy.feature, None)
+                except FeatureNotRegistered:
+                    raise ValueError(f"Feature {group_policy.feature} is not registered")
+
+            prev_ignore_limit_ratio = 0.0
+            if group_policy.limited_access:
+                prev_ignore_limit_ratio = (
+                    group_policy.limited_access.ignore_limit
+                    / group_policy.limited_access.expiry_time
+                )
```

Comment:
Could you explain this ratio? I don't totally follow

Reply:
The idea is that the ignore limit should only increase as we expand access to the group type. However, what we really want is the ratio of issues created to the time span in case the `expiry_time` is 12 hours vs 24 hours or something

Reply:
I'm not sure that we need to enforce this. I think that in general you're right and that's what people will do, but if someone for some reason wanted to do otherwise there's not too much reason to stop them.

Reply:
I would rather leave it in for safety and then if/when someone needs it we can change the code. I just don't want to see some incident because the numbers got fudged or someone got confused about ignore limit increasing vs decreasing.